datafusion/physical-plan/src/joins/utils.rs

```diff
@@ -2071,18 +2074,13 @@ mod tests {
             ),
             // No min or max (or both).
             (
-                (10, Absent, Absent, Inexact(3), Absent),
-                (10, Absent, Absent, Inexact(3), Absent),
-                None,
-            ),
-            (
-                (10, Inexact(2), Absent, Inexact(3), Absent),
-                (10, Absent, Inexact(5), Inexact(3), Absent),
+                (10, Absent, Absent, Absent, Absent),
+                (10, Absent, Absent, Absent, Absent),
```

Comment:
This became exact same as "No column level stats." input (one above).

Let's not take changing inputs in existing test cases lightly.
If they are mislabelled, perhaps let's just fix the labels